mix.exs

```diff
@@ -25,6 +25,7 @@ defmodule Onigumo.MixProject do
       # {:dep_from_hexpm, "~> 0.3.0"},
       # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"},
       {:httpoison, "~> 1.8"},
+      {:floki, "~> 0.32.0"},
```

Comment:
I’d consider creating a section with a heading describing that the following dependencies are not for the core but the tools.